### DISCUSSION TITLE
Is there compared data to other framework/library?

### DISCUSSION THREAD

**Discussion by n2ptune** [alpinejs/alpine#1064] [1. Help] [answered]
alpine looks so nice!

I want to compare to other framework/libraries for render speed, bundle size and etc....

Is there any data?

**Comment by ryansolid** [comment, marked as answer]
There is always the JS Framework Benchmark: https://github.com/krausest/js-framework-benchmark
Current results: https://krausest.github.io/js-framework-benchmark/current.html

But I think it's hard to compare with other libraries when the goal is different. They focus more on client rendering where Alpine is about augmentation.

**Reply by n2ptune** [reply]
Thanks a lot.🙏
